pkgs/top-level/all-packages.nix

```diff
@@ -717,6 +717,9 @@ in
 
   capstone = callPackage ../development/libraries/capstone { };
 
+  cataract      = callPackage ../applications/misc/cataract { };
+  cataract-head = callPackage ../applications/misc/cataract/head.nix { };
```

Comment:
Usually we use the word `unstable` to instead of `head` to denote unreleased versions.